docs/GetStarted.mdx

```diff
@@ -161,7 +176,7 @@ require_once './vendor/autoload.php';
 
 use Casbin\Enforcer;
 
-$e = new Enforcer("path/to/model.conf", "path/to/policy.csv");
+e = new Enforcer("path/to/model.conf", "path/to/policy.csv");
```

Comment:
Why? Isn't PHP var starting with $?

Reply:
Right. It was messing up with syntax highlighting, so I removed it. Forgot to add it back before committing